host-provision.sh

```diff
@@ -27,7 +27,7 @@ if [ $? != 4 ]; then
   exit
 fi
 
-GETOPT_MANDATORY="comma-separated-dbhosts:"
+GETOPT_MANDATORY="dbhost_ip:"
```

Comment:
Sorry to be a pain here, but I don't think "dbhost" is a term we're using , but rather "instance-ip-addr" (and, separately, hyphens rather than underscores in arguments).  Can we be consistent with install-oracle.sh?

Reply:
Done

Reply:
Tested successfully:
```bash
✔ ~/DriveFS/My Drive/bmaas/host_provisioning/bms-toolkit [host-provision|✚ 2] 
15:48 $ runlocalssh ./host-provision.sh --instance-ip-addr 172.16.30.1  --instance-ssh-user ansible9
Command used:
/usr/local/google/home/jcnarasimhan/DriveFS/My Drive/bmaas/host_provisioning/bms-toolkit/host-provision.sh --instance-ip-addr 172.16.30.1 --instance-ssh-user ansible9

Running with parameters from command line or environment variables:

INSTANCE_SSH_USER=ansible9
INVENTORY_FILE=172.16.30.1,
ORA_CS_HOSTS=172.16.30.1

Found Ansible at /usr/bin/ansible-playbook

Running Ansible playbook: /usr/bin/ansible-playbook -i 172.16.30.1,   host-provision.yml

PLAY [Create private public key pair locally] **********************************************************************************************************************************

TASK [Gathering Facts] *********************************************************************************************************************************************************
ok: [localhost]
...
```